WORKSPACE.bzlmod

```diff
@@ -925,163 +713,32 @@ scalatest_repositories()
-        "https://github.com/bazelbuild/rules_go/releases/download/v0.39.0/rules_go-v0.39.0.zip",
-    ],
-)
-
 # needed for cpp tests
 http_archive(
     name = "com_google_absl",
     sha256 = "987ce98f02eefbaf930d6e38ab16aa05737234d7afbab2d5c4ea7adbe50c28ed",
     strip_prefix = "abseil-cpp-20230802.1",
     urls = [
-         "https://github.com/abseil/abseil-cpp/archive/refs/tags/20230802.1.tar.gz",
+        "https://github.com/abseil/abseil-cpp/archive/refs/tags/20230802.1.tar.gz",
     ],
 )
 
-# gRPC Java
+grpc_java_version = "1.61.1"
+
 http_archive(
     name = "io_grpc_grpc_java",
-    sha256 = "3bcf6be49fc7ab8187577a5211421258cb8e6d179f46023cc82e42e3a6188e51",
-    strip_prefix = "grpc-java-1.59.0",
-    url = "https://github.com/grpc/grpc-java/archive/refs/tags/v1.59.0.tar.gz",
-)
-
-jvm_maven_import_external(
-    name = "io_netty_netty_common",
-    artifact = "io.netty:netty-common:4.1.96.Final",
-    artifact_sha256 = "da104e80db830922eaf860eb1c5e957cd1d124068253d02e9c7a7843bc66427a",
-    licenses = ["notice"],  # Apache 2.0
-    server_urls = ["https://repo1.maven.org/maven2"],
-)
-
-jvm_maven_import_external(
-    name = "io_netty_netty_transport",
-    artifact = "io.netty:netty-transport:4.1.96.Final",
-    artifact_sha256 = "8fe3afbe8b094a7b9f1eb27becf1cf017e5572343c1744d2b6040d5f331e84e3",
-    licenses = ["notice"],  # Apache 2.0
-    server_urls = ["https://repo1.maven.org/maven2"],
-)
-
-jvm_maven_import_external(
-    name = "io_netty_netty_transport_native_epoll",
-    artifact = "io.netty:netty-transport-classes-epoll:4.1.96.Final",
-    artifact_sha256 = "1591b3ea061932677dc2bab6cb7d82e8f1837a52b3c781f4daa99984ec87a9cd",
-    licenses = ["notice"],  # Apache 2.0
-    server_urls = ["https://repo1.maven.org/maven2"],
-)
-
-jvm_maven_import_external(
-    name = "io_netty_netty_transport_native_unix_common",
-    artifact = "io.netty:netty-transport-native-unix-common:4.1.96.Final",
-    artifact_sha256 = "4f96297a06a544a4cdb6fe6af8b868640f100fa96969e2196be216bd41adef13",
-    licenses = ["notice"],  # Apache 2.0
-    server_urls = ["https://repo1.maven.org/maven2"],
-)
-
-jvm_maven_import_external(
-    name = "io_netty_netty_transport_classes_kqueue",
-    artifact = "io.netty:netty-transport-classes-kqueue:4.1.96.Final",
-    artifact_sha256 = "f2f1fab3b297aee20a3922c79b548c8b4b72bb10b635375434c108ee05f29430",
-    licenses = ["notice"],  # Apache 2.0
-    server_urls = ["https://repo1.maven.org/maven2"],
-)
-
-# Usually, we'd get this from the JetBrains SDK, but the bundled one not aware of Bazel platforms,
-# so it fails on certain setups.
-jvm_maven_import_external(
-    name = "jna",
-    artifact = "net.java.dev.jna:jna:5.13.0",
-    artifact_sha256 = "66d4f819a062a51a1d5627bffc23fac55d1677f0e0a1feba144aabdd670a64bb",
-    licenses = ["notice"],  # Apache 2.0
-    server_urls = ["https://repo1.maven.org/maven2"],
+    strip_prefix = "grpc-java-{}".format(grpc_java_version),
```

Comment:
can we add `sha256`?

Reply:
Ahh, sure, I was hoping bazel would suggest it, but it didn't